cms/envs/common.py

```diff
@@ -389,6 +389,9 @@
 
     # use Django built in clickjacking protection
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+
+    # This must be last
```

Comment:
So that it will run first in the process_response chain. I think the original intent of this comment (which is copied over from the LMS common.py) was to have an inline reminder that it needs to last with duplicating the documentation that appears along with the middleware class itself. This original intent makes sense to me.